tests/Units/ConfigTest.php

```diff
@@ -1206,4 +1206,370 @@ public function testUpdateActorsDisabledConfiguration(): void
         ]);
         $this->assertCount(0, $assigned_groups);
     }
+
+    /**
+     * Test parent entity configuration inheritance
+     */
+    public function testParentEntityConfigInheritance(): void
+    {
+        $parent_entity_id = 0;
+        // Create child entity
+        $child_entity = new \Entity();
+        $child_entity_id = $child_entity->add([
+            'name' => 'Child Entity Test',
+            'entities_id' => 0, // Parent entity as parent
+        ]);
+        $this->assertGreaterThan(0, $child_entity_id);
+
+        // Configure parent entity with specific settings
+        $conf = Config::getEffectiveConfigForEntity(0);
+        $parent_config = new Config();
+        $parent_config_id = $parent_config->update([
+            'id' => $conf->getID(),
+            'entities_id' => $parent_entity_id,
+            'is_active' => 1,
+            'use_parent_entity' => 0, // This is the source config
+            'take_item_group_ticket' => 1,
+            'prevent_closure_ticket' => 1,
+            'require_technician_to_close_ticket' => 1,
+            'mandatory_task_category' => 1,
```

Comment:
Same logic here and after, when they are booleans, use `true`/`false`.
```suggestion
            'is_active' => true,
            'use_parent_entity' => false, // This is the source config
            'take_item_group_ticket' => true,
            'prevent_closure_ticket' => true,
            'require_technician_to_close_ticket' => true,
            'mandatory_task_category' => true,
```